tests/console/validate_partitioning.pm

```diff
@@ -0,0 +1,89 @@
+# Suse'S openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple LVM partition validation
```

Comment:
I'm not sure about test module name, `validate_partitioning.pm` is too generic and we don't want to use single module to validate all possible partitioning options. What's about `validate_full_lvm.pm`, or `validate_lvm_encrypt.pm` in case you are going to add some checks for the encryption.